dash_app.py

```diff
@@ -102,6 +102,8 @@ def update_figure(selected_rows):
     return app
 
 
+app = init_dashboard()
```

Comment:
@sglyon I moved `app` from a module-level variable to something only created when in __main__ to avoid creating it if this module is imported. That doesn't seem to play well with the way uwsgi wants to run it. What do you suggest?

Reply:
on other projects I've separated out the code that creates the app from the code that imports/runs it (and making it clear that importing that run file will cause it to execute) - maybe that could work here?

Reply:
You mean something like moving all the stuff in functions to another module (that can be imported without running the app) and leaving `dash_app` as a module that has the `app` as a module level variable so importing it runs the code?

Reply:
yeah that's what I was thinking!

Reply:
Can you recommend a structure for organizing this flask app in our repo?

Reply:
Addressed in https://github.com/covid-projections/covid-data-model/pull/948